.gitignore

```diff
@@ -21,6 +21,9 @@ __pycache__
 /*_data.json
 /*_embeddings.bin
 
+# pytest configuration
+pytest.ini
+
```

Comment:
Do we need this? I imagine if we had a pytest.ini file we'd probably want to have it under version control (i.e., git).